### PR TITLE
add traefik compression option, default to enable gzip.

### PR DESCRIPTION
**Pull request by sirius1024**
add traefik compress option default to enable.
to set TRAEFIK_TCP_COMPRESSION to change compression enable state.

### CODE REVIEW

File: root/opt/traefik/bin/traefik.toml.sh [hunk 1]
```diff
@@ -30,6 +31,7 @@ CATTLE_SECRET_KEY=${CATTLE_SECRET_KEY:-""}
 TRAEFIK_ENTRYPOINTS_HTTP="\
   [entryPoints.http]
   address = \":${TRAEFIK_HTTP_PORT}\"
+  compress = \":${TRAEFIK_TCP_COMPRESSION}\"
 "
 
 filelist=`ls -1 ${TRAEFIK_SSL_PATH}/*.key | cut -d"." -f1`
```
Comment (rawmind0) [inline root/opt/traefik/bin/traefik.toml.sh]: Need to add same compress option for optional https entrypoint...
```
if [ $RC -eq 0 ]; then
    TRAEFIK_ENTRYPOINTS_HTTPS="\
  [entryPoints.https]
  address = \":${TRAEFIK_HTTPS_PORT}\"
+  compress = \":${TRAEFIK_TCP_COMPRESSION}\"
```

**Comment by rawmind0** [comment]
Hi @sirius1024 ... thanks so much for your PR...

You've added compress option just for http entrypoint, but you need to add it for https entrypoint too..Please, take a look at the change request....

**Comment by sirius1024** [comment]
@rawmind0  
Sorry for that. I thought it'll work both http and https. 
I fixed it now.

**Comment by rawmind0** [comment]
Thanks @sirius1024 ....

LGTM...

**Pull request merged**